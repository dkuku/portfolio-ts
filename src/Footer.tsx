import * as React from 'react';
import { Container, Grid, Header, Segment } from 'semantic-ui-react';

const Footer: React.SFC<any> = () => {
    return (
        <div>
            <Segment inverted={true} vertical={true} style={{ padding: '5em 0em' }}>
                <Container>
                    <Grid divided={true} inverted={true} stackable={true}>
                        <Grid.Row>
                            <Grid.Column width={7}>
                                <Header as='h4' inverted={true}>Created by:</Header>
                                <p>Daniel Kukula<br/>
                                    All Rights Reserved</p>
                            </Grid.Column>
                        </Grid.Row>
                    </Grid>
                </Container>
            </Segment>
        </div>
    )
}
export default Footer;
