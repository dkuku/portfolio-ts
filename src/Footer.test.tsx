import { shallow } from 'enzyme';
import * as React from 'react';
import Footer from './Footer';

describe('Footer', ()=>{
    it('should render without crashing and match snapshot', () => {
        const component = shallow(<Footer />)
        expect(component).toMatchSnapshot();
    });
})
