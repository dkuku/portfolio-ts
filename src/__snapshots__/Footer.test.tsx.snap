// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render without crashing and match snapshot 1`] = `
<div>
  <Segment
    inverted={true}
    style={
      Object {
        "padding": "5em 0em",
      }
    }
    vertical={true}
  >
    <Container>
      <Grid
        divided={true}
        inverted={true}
        stackable={true}
      >
        <GridRow>
          <GridColumn
            width={7}
          >
            <Header
              as="h4"
              inverted={true}
            >
              Created by:
            </Header>
            <p>
              Daniel Kukula
              <br />
              All Rights Reserved
            </p>
          </GridColumn>
        </GridRow>
      </Grid>
    </Container>
  </Segment>
</div>
`;
